moveit_ros/warehouse/warehouse/CMakeLists.txt

```diff
@@ -9,9 +9,10 @@ add_library(${MOVEIT_LIB_NAME} SHARED
   src/state_storage.cpp
   src/warehouse_connector.cpp
 )
+target_include_directories(${MOVEIT_LIB_NAME} PUBLIC include)
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 ament_target_dependencies(${MOVEIT_LIB_NAME} rclcpp Boost moveit_core warehouse_ros moveit_ros_planning)
-target_link_libraries(${MOVEIT_LIB_NAME} ${OPENSSL_CRYPTO_LIBRARY})
+#target_link_libraries(${MOVEIT_LIB_NAME} ${OPENSSL_CRYPTO_LIBRARY})
```

Comment:
I left this comment in on purpose, mainly because I wasn't sure why it was in there in the first place but maybe Linux requires it.

Reply:
I'm also not sure about this. @RoboticsYY do you know if/why this was required?

Reply:
This is failing CI right now because of an undefined reference to MD5. 

```
--- stderr: moveit_ros_warehouse

/usr/bin/ld: libmoveit_warehouse.so.: undefined reference to `MD5'

collect2: error: ld returned 1 exit status

make[2]: *** [warehouse/CMakeFiles/moveit_warehouse_broadcast.dir/build.make:493: warehouse/moveit_warehouse_broadcast] Error 1

make[1]: *** [CMakeFiles/Makefile2:189: warehouse/CMakeFiles/moveit_warehouse_broadcast.dir/all] Error 2

make: *** [Makefile:141: all] Error 2

---

Failed   <<< moveit_ros_warehouse [2min 15s, exited with code 2]
```

However, I can't find any reference to MD5.